packages/@ama-sdk/schematics/README.md

```diff
@@ -79,12 +79,28 @@ If you use `Yarn2+`, you can use the following `scripts` in `package.json`:
 
 Use `generate` to (re)generate your SDK based on the content of `./swagger-spec.yaml` (make sure you have this file at the root of your project) and `upgrade:repository` to regenerate the structure of your project.
 
+#### Light SDK
+
+For SDKs that do not need to handle extensibility, the use of dictionaries or dates, a "light" version of the SDK will be generated. 
+These SDKs do not need to generate revivers, which results in a simplified version of the code and reduces the size of the bundle.
+
+If your specification file includes dates, there are multiple options for the generation of your SDK involving the global property option `stringifyDate`: 
+* By default, the option `stringifyDate` is false so dates will be generated as expected in the specifications.
+* If the dates can be of type `string`, setting the option `stringifyDate` to true will ensure the generation of a light SDK. <br>
```

Comment:
Not really true 
we can set `stringifyDate` and still have revivers because of `dictionaries`